test/ui/test_spec.js

```diff
@@ -168,6 +168,37 @@ describe('UI tests', function() {
         .then(done);
     });
 
+    it('Settings page defaults to light theme', function(done) {
+      driver.findElement(By.id('accountDropdownButton')).click()
+        .then(() => screenshotAndCompare('settingsMenu.png', 'settingsMenu', null))
+        .then(() => driver.findElement(By.id('settingsButton')).click())
+        // After clicking on the Settings menu item, wait for the dialog.
+        .then(() => driver.wait(until.elementLocated(By.id('lightThemeRadioOption'))))
+        // Wait for the fade-in to complete.
+        .then(() => driver.sleep(1000))
+        .then(() => screenshotAndCompare('settingsLightTheme.png', 'settingsLightTheme', null))
+        // Close the dialog.
+        .then(() => driver.findElement(By.xpath('//button[@data-dismiss="modal"]')).click())
```

Comment:
let's also grab a screenshot here to make sure the dialog closes after this click